src/Juvix/Prelude/Pretty.hs

```diff
@@ -166,6 +167,9 @@ indent' = indent 2
 hang' :: Doc ann -> Doc ann
 hang' = hang 2
 
+spaceOrEmpty :: Doc ann
+spaceOrEmpty = FlatAlt (Char ' ') mempty
```

Comment:
Couldn't we just use `flatAlt (pretty ' ') mempty`?

Reply:
Yes 🤦 - thanks!

Reply:
Done